fs/mount/fs_automount.c

```diff
@@ -551,6 +862,28 @@ FAR void *automount_initialize(FAR const struct automount_lower_s *lower)
       ferr("ERROR: Failed to schedule work: %d\n", ret);
     }
 
+#ifdef CONFIG_FS_AUTOMOUNTER_DRIVER
+
+  /* Initialize the new automount driver instance */
+
+  nxsem_init(&priv->exclsem, 0, 1);
+
+  /* Register driver */
+
+  sprintf(devpath, CONFIG_FS_AUTOMOUNTER_VFS_PATH "/automount%u",
```

Comment:
why not register driver at:
sprintf(devpath, CONFIG_FS_AUTOMOUNTER_VFS_PATH "%s", lower->mountpoint);

Reply:
Yes. This is a good suggestion. Thank you!

Reply:
Done. Please review. I'm just not sure should `CONFIG_FS_AUTOMOUNTER_VFS_PATH` be just `/var`, `/var/mnt` or `/var/mount`. Because now it leads to `/var/mount/mnt/sdcard0` and seems to be some king of duplication of `mount` and `mnt`.

Reply:
How about we change CONFIG_FS_AUTOMOUNTER_VFS_PATH to /var?

Reply:
Done